packages/prover/README.md

```diff
@@ -51,7 +51,7 @@ const {provider, proofProvider} = createVerifiedExecutionProvider(httpProvider,
   mutateProvider: false,
 });
 
-const web3 = new Web3(provider);
+const web3 = new Web3(httpProvider);
```

Comment:
imo not the greatest readme example but according to @nazarhussain comment here this is intended https://github.com/ChainSafe/lodestar/pull/6868#discussion_r1633145155

Reply:
Feel free to add any example you feel good.

Reply:
> Feel free to add any example you feel good.

I cannot because the types are incompatible and I don't wanna spend time fixing the prover code